.github/workflows/run_cli_on_changed_repos.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow runs on a schedule and when manually triggered. It checks for changes to the OpenNeuroDatasets-JSONLD repos, 
+# and calls a reusable workflow to run the CLI on repos that have changed *or* are new.
+# The workflow also updates the sha.txt file.
+name: Run CLI on changed repos
+
+on:
+  schedule:
+    - cron: '31 12,0 * * *'
+  workflow_dispatch:
+
+jobs:
+    get-updated-files:
+      name: Get updated JSONLD files
+      runs-on: ubuntu-latest
+      # See https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs
+      outputs:
+        dataset_list: ${{ steps.detect-changed-repos.outputs.changed_repos }}
+
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+
+        - id: detect-changed-repos
+          name: Detect changed repos
+          env:
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          working-directory: code
+          # This step runs a script that gets a list of all repos in OpenNeuroDatasets-JSONLD, and fetches each repo's latest SHA
+          # The latest repo SHAs are then compared to the recorded SHAs in code/sha.txt,
+          # and repos that (1) have different SHAs *AND* (2) have the required files for the CLI are written to a job output variable "changed_repos"
+          # that forms the list that the CLI will run on.
+          # The following kinds of repos are directly updated in the sha.txt file without being added to changed_repos (i.e., we don't run the CLI on them):
+          # - repos whose SHAs have changed, but don't have the required CLI files (TSV/JSON)
+          # - repos that do not yet have a recorded SHA (maybe a new fork)
+          # 
+          # We use syntax here to ensure that "repos" is a multiline string, where each line represents one repo
+          # See: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings
```

Comment:
:tada: 

But please move this down to where you construct the multi-line GITHUB_OUTPUT, and also update `"repos"` to `"changed_repos"` to make clearer what's referred to here. This is pretty complex stuff, so let's be kind to future us